sdk/apimanagement/api-management-custom-widgets-tools/review/api-management-custom-widgets-tools.api.md

```diff
@@ -0,0 +1,90 @@
+## API Report File for "@azure/api-management-custom-widgets-tools"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export const ASK_FOR_SECRETS_MESSAGE_KEY = "askForSecretsMSAPIM";
+
+// @public
+export const askForSecrets: (targetModule: TTargetModule, { origin: targetOrigin, instanceId, environment }: TPortalData) => Promise<TSecrets>;
+
+// @public
+export function buildBlobConfigSrc(name: string): string;
+
+// @public
+export function buildBlobDataSrc(name: string): string;
+
+// @public
+export function buildOnChange<TValues extends TValuesCommon>(valuesDefault: TValues): TOnChange<TValues>;
+
+// @public
+export const CONFIG_FILE_NAME = "config.msapim.json";
+
+// @public
+export const CONFIGS_FOLDER = "configs";
+
+// @public
+export const DATA_FOLDER = "data";
+
+// @public
+export function deployNodeJS(serviceInformation: TServiceInformation, name: string, fallbackConfigPath?: string): Promise<void>;
+
+// @public
+export const EDITOR_DATA_KEY = "editorData";
+
+// @public
+export function getEditorData<TValues extends TValuesCommon>(valuesDefault: TValues): TEditorData<TValues>;
+
+// @public
+export function getEditorValues<TValues extends TValuesCommon>(valuesDefault: TValues): TValues;
+
+// @public
+export const ON_CHANGE_MESSAGE_KEY = "customInputValueChangedMSAPIM";
+
+// @public
+export function onChangeWithOrigin<TValues extends TValuesCommon>(origin: TEditorData<TValues>["origin"], instanceId: TEditorData<TValues>["instanceId"], values: TValues): void;
+
+// @public
+export const ROOT = "custom-widgets";
+
+// @public
+export interface TEditorData<TValues extends TValuesCommon> extends TPortalData {
+    values: TValues;
+}
+
+// @public
+export type TEnvironment = "development" | "publishing" | "runtime" | "error";
+
+// @public
+export type TOnChange<TValues extends TValuesCommon> = (values: Partial<TValues>) => void;
+
+// @public
+export interface TPortalData {
+    environment: TEnvironment;
+    instanceId: string;
+    origin: string;
+}
+
+// @public
+export type TSecrets = {
+    token: string;
```

Comment:
is this an accessToken?

Reply:
no

Reply:
Okay, what kind of token is this? I'd recommend to be explicit in the property name to make sure users know what kind of token they are dealing with. This is important to give more infor in case they are thinking about logging or sharing

Reply:
This is a user-scoped token used to access data plane API. So I guess we can name it accessToken if you really want to.